### ISSUE TITLE
Im having problems starting The Vanilla Beyond...

### ISSUE THREAD

**Issue by sam230001** [reteo/The-Vanilla-Beyond#1]
ill click play and it closes the main tab but then reopens its... nothing happens

**Comment by reteo** [comment]
What I'm going to have you do is close ATLauncher, and then open it fresh, and try again.  If it crashes again (and it does sound like the game crashed), then click "Upload Log" at the bottom of the log window.  Then paste the URL into a new comment in this thread.  This should allow me to see what's happening with the game, so I can pinpoint the problem you're having with it.

**Comment by Soundy777** [comment]
I'm suffering from the same issue as Sam. When I try to launch the instance it closes the launcher for a moment and then reopens the launcher. The console is reporting a series of errors. I've tried the pack with no optional mods, all optional mods & only some, each time it crashes in the same manner. 
Here is an upload of the log: http://paste.atlauncher.com/view/d7b2ee3a
During the session the log details all I did was to install the pack with the mods selected that I wanted (not all but most) and then try to launch the instance.
In the meantime I'm happily playing in several other mod packs but yours looks really interesting and suits my needs much better than the rest, I'd really appreciate it if you could resolve this issue.
If you need any more information I'll try my best to provide it & if you do manage to resolve the issue let me know & I'll happily test it to see if it works. Thanks in advance for your time.
An additional note is that I'm running java version 1.7.0_79-b15 in case that matters (I'm reluctant to update my java as 1.7 is required for some of the other mod packs & for projects outside of playing minecraft).

**Comment by reteo** [comment]
Actually, that probably is the problem.  JMOD, the tool I use to make most of the customizations, needs Java (1.)8 in order to work.

**Comment by Soundy777** [comment]
In that case I'll bit the bullet & update to 1.8. I'll be done that
sometime later this evening (or possibly tomorrow) & will let you know if
it resolves the issue. Thanks again for your help.

**Comment by reteo** [comment]
No problem!  Hope it works for you!

**Comment by Soundy777** [comment]
Yep. Updating to 1.8.0_101-b13 solved the issue causing the instance to fail to launch & return to the launcher. Thanks again for your help & I hope this information helps other too. 
Important side note is that when you update java the ATLauncher doesn't automatically switch to using the newer version & requires you go into settings and point it to the new java installation.

**Comment by reteo** [comment]
That's very good to know.  You might want to bring this up with the [ATLauncher Devs](https://github.com/ATLauncher/ATLauncher/issues).
